### DISCUSSION TITLE
Make "${mailscript:MAILSCRIPTNAME} a clickable link

### DISCUSSION THREAD

**Discussion by HeliosMon** [arnoudkooi/SN-Utils#234] [Q&A] [answered]
Hey Arnoud, 

while creating a notification, I always have to copy the mailscript name and then go the mailscript table and search it. (Well, I could probably write a slashcommand for it)

Since a Mailscript call in a notification always starts with "${mailscript:" I was wondering if it's possible to make it clickable via SN-Utils or show up above the Textfield as a reference to click.

What do you think?

**Comment by arnoudkooi** [comment, marked as answer]
Isn't that what the Related Link UI action "Show Notification Scripts" is for?

**Comment by HeliosMon** [comment]
@arnoudkooi This is kinda embarassing. You're correct. Thanks!
